app/presenters/collection_presenter.rb

```diff
@@ -63,4 +63,19 @@ def license_label
   def review_workflow_status
     collection.review_enabled ? 'On' : 'Off'
   end
+
+  def purl_link
+    # No druid yet, so there's no PURL link yet either. Collection is likely still depositing.
+    return if druid.blank?
+
+    link_to(nil, Sdr::Purl.from_druid(druid:), target: '_blank', rel: 'noopener')
```

Comment:
Extract an external link into a helper to avoid having to dupe the target and rel?

Reply:
I'm not sure where the target and rel are being duped--could you explain where this might be extracted to?

Reply:
https://github.com/sul-dlss/hungry-hungry-hippo/issues/529